README.md

```diff
@@ -12,7 +12,7 @@
   <a href="https://warp.dev/blog">Blog</a>
 </p>
 
-<a href="https://www.youtube.com/watch?v=T7R8lvvBgOI">
+<a href="https://app.warp.dev/get_warp">
```

Comment:
i think we should still keep the demo link as before

Reply:
got it